artemis/reporting/severity.py

```diff
@@ -28,6 +28,7 @@ class Severity(str, Enum):
     ReportType("exposed_configuration_file"): Severity.HIGH,
     ReportType("exposed_sql_dump"): Severity.HIGH,
     ReportType("exposed_ssh_with_easy_password"): Severity.HIGH,
+    ReportType("exposed_to_sql_injection"): Severity.HIGH,
```

Comment:
call it sql_injection:core (this is ugly but we need to distinguish from sql injection in extra)

Reply:
Done